mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -1602,7 +1602,9 @@ static void processPassthroughAttrs(llvm::Function *func, LLVMFuncOp funcOp) {
     // explicit attribute.
     // Also skip the vscale_range, it is also an explicit attribute.
     if (attrName == "aarch64_pstate_sm_enabled" ||
-        attrName == "aarch64_pstate_sm_body" || attrName == "vscale_range")
+        attrName == "aarch64_pstate_sm_body" || 
```

Comment:
Let's maybe update the comment above to something more generic not enumerating all the attributes (since this will be outdated once we add the next explicit attribute. What about putting something like this:

    // Skip all function attributes that have an explicit attribute in the 
    // LLVMFuncOp argument list and thus should not be stored in the 
    // passthrough attribute dictionary.

Reply:
Note that this has changed with a recently landed commit, so maybe rebase first.

Reply:
Ah was not aware of that. Rebasing will resolve the comment then.